Lib/test/test_repl.py

```diff
@@ -0,0 +1,29 @@
+"""Test the interactive interpreter."""
+
+import unittest
+from textwrap import dedent
+from test.support import run_pty, SuppressCrashReport
+
+class TestInteractiveInterpreter(unittest.TestCase):
+
+    def test_no_memory(self):
```

Comment:
Please decorate the test with @support.cpython_only, or use support.import_module('_testcapi'), to skip the test on Python implementations without _testcapi.